FMPFeedbackForm/FMPFeedbackForm.h

```diff
@@ -6,8 +6,8 @@
 //  Copyright © 2020 MacPaw. All rights reserved.
 //
 
-#import <FMPFeedbackForm/FMPFeedbackParameter.h>
-#import <FMPFeedbackForm/FMPFeedbackSender.h>
-#import <FMPFeedbackForm/FMPZendeskFeedbackSender.h>
-#import <FMPFeedbackForm/FMPFeedbackController.h>
-#import <FMPFeedbackForm/FMPInterfaceSettings.h>
+#import "FMPFeedbackParameter.h"
+#import "FMPFeedbackSender.h"
+#import "FMPZendeskFeedbackSender.h"
+#import "FMPFeedbackController.h"
+#import "FMPInterfaceSettings.h"
```

Comment:
this pobably, should go under #if SWIFT_Package, since there's a requirement having Module imports using angular brackets https://developer.apple.com/forums/thread/651941

Reply:
In my testing, I did not see any build errors or warnings with this setting, however I guess this does not hurt, added conditional imports.